src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -166,6 +108,75 @@ namespace Platform
         return exePath;
     }
 
+    static std::vector<std::string> GetPathSearchPrefixes()
+    {
+        std::vector<std::string> prefixes;
+        auto exePath = Platform::GetCurrentExecutablePath();
+        prefixes.push_back(Path::GetDirectory(exePath));
+        prefixes.push_back(GetCurrentWorkingDirectory());
+        prefixes.push_back("/");
+        return prefixes;
+    }
+
+    std::string GetDocsPath()
+    {
+        const auto prefixes = GetPathSearchPrefixes();
+        static const utf8* searchLocations[] = {
+            "./doc",
+            "/usr/share/doc/openrct2",
```

Comment:
Unless you have an empty prefix in that list above, you will get `//usr/share/doc/openrct2`.

Reply:
should be addressed now